Analysis/include/SampleDescriptor.h

```diff
@@ -20,11 +20,14 @@ struct AnalyzerSetup {
     Period period;
     SignalMode mode;
     QCDmethod qcd_method;
+    EventRegion qcd_shape;
     DiscriminatorWP tauID_wp;
     std::vector<std::string> final_variables;
     std::vector<double> pt_sel_bins;
-    bool apply_mass_cut{false}, apply_os_cut{true}, apply_iso_cut{true};
+    bool apply_mass_cut{false}, apply_os_cut{true};
```

Comment:
Also `apply_mass_cut` and `apply_os_cut` makes no sense for the analyzer. Please, check that they are never used and remove them.